supplementary_style_guide/glossary_terms_conventions/general_conventions/h.adoc

```diff
@@ -298,6 +309,17 @@ Hidden replicas are primarily designed for services such as backups, bulk import
 
 *See also*:
 
+[[hypershift-operator]]
+==== image:images/yes.png[yes] HyperShift Operator (noun)
+*Description*: The _HyperShift Operator_ manages the lifecycle of hosted clusters that are representeed by the `HostedCluster` API. The HyperShift Operator along with the `hcp` command-line interface (CLI) are components of hosted control planes.
```

Comment:
```suggestion
*Description*: The _HyperShift Operator_ manages the lifecycle of hosted clusters that are represented by the `HostedCluster` API. The HyperShift Operator along with the `hcp` command-line interface (CLI) are components of hosted control planes.
```

Reply:
Also, just a thought. Would the word 'and' be a good alternative to 'along with'? for conciseness and translation/international readers. Disregard this if it changes the meaning.

`The HyperShift Operator and the `hcp` command-line interface (CLI) are components of hosted control planes.`